etc/module.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="../../../../lib/internal/Magento/Framework/Module/etc/module.xsd">
-    <module name="VinaiKopp_EavOptionSetup" setup_version="0.1.0">
+    <module name="VinaiKopp_EavOptionSetup" setup_version="1.0.0">
```

Comment:
I would prefer to keep the version at `0.1.0`. No need to raise the version to `1.0` which people can easily mistake to have a special meaning.